eve/workers/centos7_terraform/requirements.sh

```diff
@@ -0,0 +1,9 @@
+TERRAFORM_VERSION=0.11.8
+
+
+yum install -y epel-release unzip
+curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin/
+yum install -y haveged
+systemctl start haveged
+sudo -u eve ssh-keygen -t rsa -b 2048 -N '' -f /home/eve/.ssh/id_rsa
```

Comment:
Would it make sense to *not* use `~/.ssh/id_rsa` but some custom location instead, and let Terraform use that one (it's a variable, right)?

Reply:
Yes possible indeed, but what's the purpose of a custom location ? What's wrong with ~/.ssh/id_rsa ?

Reply:
What is Eve gets some more dedicated support for SSH access to other nodes/VMs/... and decides to use `~/.ssh`?

Reply:
Then that's cool, we don't need to create one and just start using it ? (That's a real newbie answer, I know, maybe I'm wrong, let's ask @tcarmet )